packages/sdk/__tests__/resources/manifestSnapshotMap.ts

```diff
@@ -1,6 +1,58 @@
 import type { Stats, ModuleInfo } from '../../src/types';
 
 const manifest: { [key: string]: Stats } = {
+  ssrAppManifest: {
+    id: '@mf/ssr-manifest-provider',
+    name: '@mf/ssr-manifest-provider',
+    metaData: {
+      type: 'global',
+      pluginVersion: '0.0.1',
+      name: '@mf/ssr-manifest-provider',
+      buildInfo: {
+        buildVersion: '1.0.0.1517',
+        buildName: 'ssr-manifest-provider',
+      },
+      ssrRemoteEntry: {
+        name: 'federation-remote-entry.496985d3.js',
+        path: 'server',
+        type: 'commonjs-module',
```

Comment:
Why not commonjs but commonjs module?

Reply:
It support all the output.libraryTarget value https://webpack.js.org/configuration/output/#outputlibrarytarget